kyuubi-server/src/test/scala/org/apache/kyuubi/operation/PlanOnlyOperationSuite.scala

```diff
@@ -114,6 +114,24 @@ class PlanOnlyOperationSuite extends WithKyuubiServer with HiveJDBCTestHelper {
     }
   }
 
+  test("kyuubi #3214: Plan only mode with an incorrect value") {
+    withSessionConf()(Map(KyuubiConf.OPERATION_PLAN_ONLY_MODE.key -> "parse"))(Map.empty) {
+      withJdbcStatement() { statement =>
+        statement.executeQuery(s"set ${KyuubiConf.OPERATION_PLAN_ONLY_MODE.key}=parser")
+        try {
+          statement.executeQuery("select 1")
+        } catch {
+          case e: Exception =>
+            assert(e.getMessage.contains("The operation mode UNKNOWN doesn't support"))
+        }
+        statement.executeQuery(s"set ${KyuubiConf.OPERATION_PLAN_ONLY_MODE.key}=parse")
```

Comment:
this test works because of `kyuubi.operation.plan.only.excludes`.

So this is actually a side-effect and indeterministic.